uninstall.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/sh
+
+# Uninstall script for cake-autorate
+#
+# See https://github.com/lynxthecat/cake-autorate for more details
+
+# This needs to be encapsulated into a function so that we are sure that
+# sh reads all the contents of the shell file before we potentially erase it.
+#
+# Otherwise the read operation might fail and it won't be able to proceed with
+# the script as expected.
+main() {
+	# Set correctness options
+	set -eu
+
+	# Stop cake-autorate before continueing
+	if [ -x /etc/init.d/cake-autorate ]
+	then
+		/etc/init.d/cake-autorate stop || true
+	fi
+	rm -f /etc/init.d/cake-autorate /etc/rc.d/*cake-autorate
+
+	# Check if an instance of cake-autorate is already running and exit if so
+	if [ -d /var/run/cake-autorate ]
+	then
+		printf "cake-autorate is already running - exiting\n" >&2
+		printf "If you want to install a new version, stop the service first\n" >&2
```

Comment:
```suggestion
		printf "If you want to install a new version, first stop any running instance of cake-autorate\n" >&2
```

Reply:
Well, its an uninstaller. That disclaimer needs to be changed.